drivers/at86rf215/Kconfig

```
@@ -38,4 +38,95 @@ config AT86RF215_TRIM_VAL
         26 MHz the best.
         For more information Refer Table 6-25 TRIM in Datasheet
 
+choice
+    prompt "Default Modulation"
+
+config AT86RF215_DEFAULT_LEGACY_OQPSK
+    bool "legacy O-QPSK"
+    help
+        O-QPSK compatible with IEEE 802.15.4-2003 devices
+
+config AT86RF215_DEFAULT_MR_OQPSK
+    bool "MR-O-QPSK"
+    help
+        MR-O-QPSK according to IEEE 802.15.4g
+
+config AT86RF215_DEFAULT_MR_OFDM
+    bool "MR-OFDM"
+    help
+        MR-O-OFDM according to IEEE 802.15.4g
+
+endchoice
+
+menu "O-QPSK (802.15.4) configuration"
+
```

I think it would be good to make these menus depend on the correspondent modulation being selected, so we only show relevant configurations:
```suggestion
menu "O-QPSK (802.15.4) configuration"
    depends on AT86RF215_DEFAULT_LEGACY_OQPSK
```

---

good idea, this only affects the defaults.

All modulations are still available at run-time, but when switching at run-time the last configured parameters will be used, not the default ones.